cmd/apps/helm_operator_app.go

```diff
@@ -0,0 +1,146 @@
+		SilenceUsage: true,
+	}
+
+	operator.Flags().StringP("namespace", "n", "default", "The namespace used for installation")
+	operator.Flags().Bool("update-repo", true, "Update the helm repo")
+	operator.Flags().Bool("helm3", true, "Use helm3, if set to false uses helm2")
+
+	operator.RunE = func(command *cobra.Command, args []string) error {
+		kubeConfigPath := getDefaultKubeconfig()
+
+		if command.Flags().Changed("kubeconfig") {
+			kubeConfigPath, _ = command.Flags().GetString("kubeconfig")
+		}
+
+		updateRepo, _ := operator.Flags().GetBool("update-repo")
+
+		fmt.Printf("Using kubeconfig: %s\n", kubeConfigPath)
+		helm3, _ := command.Flags().GetBool("helm3")
+
+		userPath, err := config.InitUserDir()
+		if err != nil {
+			return err
+		}
+		namespace, _ := command.Flags().GetString("namespace")
+		if namespace != "default" {
+			return fmt.Errorf(`to override the "default", install via tiller`)
+		}
+
+		clientArch, clientOS := env.GetClientArch()
+
+		fmt.Printf("Client: %s, %s\n", clientArch, clientOS)
+		log.Printf("User dir established as: %s\n", userPath)
+
+		os.Setenv("HELM_HOME", path.Join(userPath, ".helm"))
+
+		_, err = helm.TryDownloadHelm(userPath, clientArch, clientOS, helm3)
+		if err != nil {
+			return err
+		}
+
+		err = addHelmRepo("fluxcd", "https://charts.fluxcd.io", helm3)
+		if err != nil {
+			return err
+		}
+
+		if updateRepo {
+			err = updateHelmRepos(helm3)
+			if err != nil {
+				return err
+			}
+		}
+
+		chartPath := path.Join(os.TempDir(), "charts")
+		err = fetchChart(chartPath, "fluxcd/helm-operator", helm3)
+
+		if err != nil {
+			return err
+		}
+
+		overrides := map[string]string{}
+		// the helm chart always installs the crds
+		//overrides["createCRD"] = "true"
+		overrides["helm.versions"] = "v3 "
+
+		arch := getNodeArchitecture()
+
+		fmt.Printf("Node architecture: %q\n", arch)
+
+		// Only so long until we get arm support
+		if arch == "arm64" {
+			overrides["image.repository"] = "docker.io/onedr0p/helm-operator"
+			overrides["image.tag"] = "latest"
+		}
+
+		fmt.Println("Chart path: ", chartPath)
+
+		ns := "default"
+
+		log.Printf("Applying CRD: CRDs will be applied by the helm chart\n")
+
+		if helm3 {
+			outputPath := path.Join(chartPath, "helm-operator")
+
+			err := helm3Upgrade(outputPath, "fluxcd/helm-operator", ns,
+				"values.yaml",
+				"",
+				overrides,
+			)
+
+			if err != nil {
+				return err
+			}
+		} else {
+			outputPath := path.Join(chartPath, "helm-operator/rendered")
+
+			err = templateChart(chartPath,
+				"helm-operator",
+				ns,
+				outputPath,
+				"values.yaml",
+				"",
+				overrides)
+
+			if err != nil {
+				return err
+			}
+
+			err = kubectl("apply", "-R", "-f", outputPath)
+
+			if err != nil {
+				return err
+			}
+		}
+
+		fmt.Println(helmOperatorInstallMsg)
+		return nil
+	}
+
+	return operator
+}
+
+const helmOperatorInstallMsg = `# The helm-operator has been configured.
+# By default it will use the ssh-key from k3sup app install fluxcd
```

Comment:
Rename k3sup to arkade please

Can you give the testing instructions you gave in the PR, but in the output text, so that people can run that and see what it looks like?

Perhaps a YAML file / CR they can apply?